test/libsolidity/smtCheckerTests/types/fixed_bytes_access_2.sol

```diff
@@ -0,0 +1,11 @@
+pragma experimental SMTChecker;
+contract C {
+	function f(bytes calldata x, uint y) external pure {
+		x[8][5%5];
+		x[8][5%y];
+	}
+}
+// ----
+// Warning 7989: (101-110): Assertion checker does not yet support index accessing fixed bytes.
```

Comment:
Strictly speaking this warning should refer to array type (bytes array) instead of fixed bytes, right?

Reply:
Nop, accessing `bytes` is fine, the issue here is accessing an index of a `FixedBytes` type which is valid in Solidity but the SMTChecker doesn't support yet.

Reply:
Is the `5%5` necessary in the test case, or could we simply do `0` for readability's sake. Fine with it this way if you are.

Reply:
not necessary, can just be 0